### PR TITLE
Addopssencryptkey

### PR DESCRIPTION
**Pull request by jshum2479**

### CODE REVIEW

File: core/src/main/python/wlsdeploy/util/wlst_helper.py [hunk 1]
```diff
@@ -1395,3 +1395,16 @@ def reopen_offline(domain_home):
...
     _logger.exiting(class_name=_class_name, method_name=_method_name)
+
+
+def set_shared_secret_store_with_password(wallet_path, password):
+    """
+    Set opss store password
+    :param wallet_path: opss extracted wallet
+    :param password:  opss store extraction time password
+    """
+    _method_name = 'setSharedSecretStoreWithPassword'
```
Comment (rakillen) [inline core/src/main/python/wlsdeploy/util/wlst_helper.py]: doesn't match method name set_shared_secret_store_with_password

File: core/src/main/python/wlsdeploy/tool/create/domain_creator.py [hunk 2]
```diff
@@ -1128,3 +1134,51 @@ def __create_boot_dot_properties(self):
             ostream.close()
         self.logger.exiting(class_name=self.__class_name, method_name=_method_name)
         return
+
+    def __configure_opss_secrets(self):
```
Comment (ddsharpe) [inline core/src/main/python/wlsdeploy/tool/create/domain_creator.py]: Dont we need to screen off the online use case?  You cannot update the secret in online mode, can you?

Comment (jshum2479) [reply core/src/main/python/wlsdeploy/tool/create/domain_creator.py]: It's for offline , not online

Comment (jshum2479) [reply core/src/main/python/wlsdeploy/tool/create/domain_creator.py]: and during creation.

Comment (jshum2479) [reply core/src/main/python/wlsdeploy/tool/create/domain_creator.py]: i.e. it's not even in the code path for online

**Comment by jshum2479** [comment]
@CarolynRountree   @ddsharpe 

1.  Should this be documented and where
2.  I am trying a place to move the common python code.
3.  There is no discovery for now.

**Pull request merged**